RearrangeScreensModule/src/test/java/com/siri/RearrangeScreens/ExampleUnitTest.kt

```diff
@@ -0,0 +1,16 @@
+package com.siri.RearrangeScreens
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
+    @Test
+    fun addition_isCorrect() {
+        assertEquals(4, 2 + 2)
+    }
+}
```

Comment:
Ah, not a real test. You're keeping it as just an example?  (It's okay if you are; just want to understand:)

Reply:
That was just an example